test/Snowflake_Tests/src/Snowflake_Spec.enso

```diff
@@ -505,10 +505,10 @@ snowflake_specific_spec suite_builder default_connection db_name setup =
             do_round -1.2222222222222235 15 . should_equal -1.222222222222224
 
         group_builder.specify "Can round correctly near the precision limit, using banker's rounding" <|
-            do_round 1.2222222222222225 15 use_bankers=True . should_equal 1.222222222222222
-            do_round -1.2222222222222225 15 use_bankers=True . should_equal -1.222222222222222
-            do_round 1.2222222222222235 15 use_bankers=True . should_equal 1.222222222222224
-            do_round -1.2222222222222235 15 use_bankers=True . should_equal -1.222222222222224
```

Comment:
The previous tests should be kept and executed, otherwise this is a breaking change. Adding a label, if breakage is intended.